operations/customers.md

```diff
@@ -505,9 +522,17 @@ Updates personal information of a customer. Note that if any of the fields is le
 | `Visa` | [Document](customers.md#document) | optional | New visa details. |
 | `DriversLicense` | [Document](customers.md#document) | optional | New drivers license details. |
 | `Address` | [Address parameters](customers.md#address-parameters)  | optional | New address details. |
+| `LegalIdentifiers` | [Legal identifiers update](#legal-identifiers) | optional | Update to legal identifiers. |
```

Comment:
The link is most probably broken as it does not link to the object below.

Reply:
It refers to the one for creation 🤦